charts/pwa/templates/ingress.yaml

```diff
@@ -1,131 +1,43 @@
+{{- define "pwa-ingress.name" -}}
+{{ regexReplaceAll "\\W+" .host "" }}
+{{- end -}}
 {{- if .Values.ingress.enabled -}}
-{{- $fullName := include "pwa-ingress.service" . -}}
-{{- $svcPort := .Values.cache.service.port -}}
-{{- if and .Values.ingress.className (not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion)) }}
-  {{- if not (hasKey .Values.ingress.annotations "kubernetes.io/ingress.class") }}
-  {{- $_ := set .Values.ingress.annotations "kubernetes.io/ingress.class" .Values.ingress.className}}
-  {{- end }}
-{{- end }}
-{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
+{{ range $index, $value := .Values.ingress.hosts }}
```

Comment:
Imagine you would have 300 hosts/channels and you needed to define IP-whitelisting with up to 80 IPs for 180 of those hosts in the values file. That is the usecase we had in mind when implementing this and that is why we decided to simply go with 2 ingresses (1 for non-live with its IP whitelistings and 1 for live without)